js/react_native/e2e/android/app/src/debug/java/com/example/reactnativeonnxruntimemodule/ReactNativeFlipper.java

```diff
@@ -4,7 +4,7 @@
  * <p>This source code is licensed under the MIT license found in the LICENSE file in the root
  * directory of this source tree.
  */
-package com.reactnativeonnxruntimemodule;
+package com.example.reactnativeonnxruntimemodule;
```

Comment:
Why do we use the "example.com" domain?

Reply:
@edgchen1

Reply:
I'm not sure. my guess is that it is because this is an E2E test application.

Reply:
I just findout this file was generated by react-native, and it will be removed in. 0.73